src/schema/v2/artwork/index.ts

```diff
@@ -1124,8 +1124,11 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         },
       },
       priceListed: {
-        type: GraphQLFloat,
-        resolve: ({ price_listed }) => price_listed,
+        type: Money,
+        resolve: ({ price_listed: price_listed, price_currency: currency }) => {
+          const cents = price_listed * 100
+          return { cents, currency }
```

Comment:
For `display`, the idea is to use the existing `internalDisplayPrice`.

Reply:
👍  I'm fine to skip implementing `display` now if it's not needed yet.